e2e_test/source/basic/kafka.slt

```diff
@@ -478,6 +478,18 @@ create table source_with_rdkafka_props (v1 int, v2 varchar) with (
   properties.queued.max.messages.kbytes = 65536
 ) FORMAT PLAIN ENCODE JSON
 
+statement ok
+CREATE TABLE debezium_ignore_key (
+  order_id int primary key
+) with (
+  connector = 'kafka',
+  topic = 'debezium_mess_key',
+  kafka.brokers = 'message_queue:29092',
+  kafka.scan.startup.mode = 'earliest')
+FORMAT DEBEZIUM ENCODE JSON (
+  ignore_key = 'true'
```

Comment:
Is it possible to put this in the `with` statement? This may happen for debezium avro as well.

Reply:
It is an option for `format` rather than the connector, not suitable for putting it in with clause. 

> This may happen for debezium avro as well.

Yes, this option can also apply to debezium avro.